### PR TITLE
Fix some typo

### PR DESCRIPTION
**Pull request by HackToday**
Signed-off-by: Kai Qiang Wu(Kennan) <wkqwu@cn.ibm.com>

### CODE REVIEW

File: docs/swarm_at_scale/deploy-infra.md [hunk 1]
```diff
@@ -264,7 +264,7 @@ in Step 4.
     ```
 
     This error usually means you weren't starting the `docker run` from the same
-    `config` directory where the `config.toml` fie is. If you run the coammand
+    `config` directory where the `config.toml` fie is. If you run the command
```
Comment (MHBauer) [inline docs/swarm_at_scale/deploy-infra.md]: fie -> file

**Comment by HackToday** [comment]
ping @MHBauer and @dongluochen

**Comment by vieux** [comment]
LGTM thanks @HackToday

**Comment by MHBauer** [comment]
Ya good. Thanks for the update @HackToday.

**Pull request merged**